### DISCUSSION TITLE
method to disable go guardian omada dns ripoff

### DISCUSSION THREAD

**Discussion by 88411** [3kh0/ext-remover#1410] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
**EDIT I just realized that this is literally a copy of omada dns. At the time I knew that there were other dns's already out there however I didn't realize that it was already known that restarting clears go guardian cache. I will leave this post up anyways tho just bc it is funny **

Maybe someone can give this a name and improve it. 

Anyways here are the basic steps. ( based off of my last post)

1.) connect to my dns server

Set your dns to 45.90.28.127
Then click the syncing link (insert link)

2.) wipe goguardian memory/cache? (Aka give go guardian dementia so it forgets how to do its job)

Type in the url bar chrome://restart

You can also just regular restart but on the latest update the chrome://reatart doesn't close any of your tabs.

You can also set the url as a bookmark by opening a new tab, saving it as a bookmark. Then right clicking and setting the bookmarks url to chrome://restart

As long as you are on the dns and it is properly synced you can turn your Chromebook on or off and it will continue to work      

To my knowledge here is how I think it works:

Basically you restart the Chromebook and either all chrome extensions memory/cache gets wiped or just go guardians.

Then bc you are on my dns, go guardian can't redownload the policies/ data that telling it what to block.

Or in simple terms you get go guardian to forget how to do its job and then make it so I can't remember how. 

You guys can correct me if I'm
Wrong lol

**Comment by awsome-webdev** [comment]
wow that is alot easier than exthanger
>but srry
-how do i change my dns?
-where do i clear my memory/cache for goguardian

**Reply by 88411** [reply]
The clear go guardian cache type in the url bar chrome://restart but make dns is on before you do. Change dns by clicking on your network and scrolling down. Set the bottom three to 0.0.0.0 the the first on the 45.90.28.127

**Reply by awsome-webdev** [reply]
thank you!
